app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
You have an extra `else` statement here. According to the checklist, it is better to avoid `else` when the `if` clause ends with a `return` statement. This makes the code easier to read and understand. Here's how you can re-write this piece of code:

python
if self.is_hungry:
    print(f"Eating {self.appetite} food points...")
    self.is_hungry = False
    return self.appetite
return 0
```